packages/cli/__tests__/zosfiles/__unit__/edit/Edit.utils.unit.test.ts

```diff
@@ -206,41 +208,7 @@ describe("Files Edit Utilities", () => {
             });
         });
     });
-    describe("localDownload()", () => {
```

Comment:
Should this test suite be removed? It seems to be handling tests for downloading files from the mainframe.

Reply:
Agree that these tests probably shouldn't be removed.

The [v3 PR to add edit options](https://github.com/zowe/zowe-cli/pull/2027) added 8 tests and removed 0.

I would expect this PR to do the same, but currently it adds 4 tests and removes 2.

Reply:
tests are present, view file to and search for '(did not write to temp location)'